src/javascripts/application.mjs

```diff
@@ -104,3 +111,52 @@ if (campaignCookieBanner) {
     subtree: true
   })
 }
+
+const serviceNavigationHeader = document.querySelector(
+  '.govuk-service-navigation'
+)
+
+if (
+  serviceNavigationHeader.hasAttribute('data-govuk-service-navigation-init')
+) {
+  const subNavTreeParents = serviceNavigationHeader.querySelectorAll('li')
```

Comment:
**suggestion** We can access the links and use `insertAdjacentElement` to inject the sub navigation and button.